src/main/java/net/snowflake/client/core/SFResultSetMetaData.java

```diff
@@ -353,7 +353,8 @@ public int getColumnType(int column) throws SFException {
       externalColumnType = Types.TIMESTAMP;
     }
     if (internalColumnType == SnowflakeUtil.EXTRA_TYPES_TIMESTAMP_TZ) {
-      externalColumnType = Types.TIMESTAMP_WITH_TIMEZONE;
+      externalColumnType =
```

Comment:
NIT: Add comments to explain the reason to return TIMESTAMP.

Reply:
I feel like the function name is self-explanatory, because the code basically says if return timestamp with timezone is enabled, return timestamp with timezone, otherwise return timestamp. So the comments in the function getEnableReturnTimestampWithTimeZone should be enough.